### PR TITLE
[new] [author] Add Wing@0.1.4 w/ npm autoupdate

### PR DESCRIPTION
**Pull request by KingPixil**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/KingPixil/wing
 * Official website (optional, not the repository): http://usewing.ml
 * NPM package url (optional): https://www.npmjs.com/package/wingcss
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 115
   - Count of watchers: 6
   - Count of forks: 12
 * NPM download stats (optional):
   - Downloads in the last day: 2
   - Downloads in the last week: 65
   - Downloads in the last month: 275

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

* I used Github's web interface, as locally cloning cdnjs would take over 2 hours.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@KingPixil let's use npm auto-update and add some keywords, thanks.

@cdnjs/intern2 please also help review this PR as your practice, thanks.

**Comment by KingPixil** [comment]
@PeterDaveHello So I cannot use Git autoupdate?
Sorry I am a noob at this :stuck_out_tongue_winking_eye:

**Comment by PeterDaveHello** [comment]
@KingPixil It's okay but we prefer npm, no sorry, thanks for your contribution.

**Comment by KingPixil** [comment]
@PeterDaveHello Just asking, I can use NPM though.

**Comment by KingPixil** [comment]
@PeterDaveHello Added keywords and npm auto-update.

File: ajax/libs/wingcss/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "framework",
+    "semantic"
+  ],
+  "npmName": "wingcss",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "wing.min.css",
+        "wing.css"
```
Comment (pvnr0082t) [inline ajax/libs/wingcss/package.json]: @KingPixil  Let's use `**/*` here.

**Comment by KingPixil** [comment]
@pvnr0082t Got it

**Comment by KingPixil** [comment]
@pvnr0082t Just to make sure, it will select the same files, right?

**Comment by pvnr0082t** [comment]
@KingPixil Right!! :facepunch:

**Comment by PeterDaveHello** [comment]
Thanks @KingPixil

**Pull request merged**